core/settings.py

```diff
@@ -64,8 +64,10 @@
     "django.contrib.staticfiles",
     "django_extensions",
 
-    'home',
-    'theme_pixel'
+     'home',
+    'theme_pixel',
+    'rest_framework',  
+    'drf_yasg', 
```

Comment:
Please add drf-yasg and its version in the requirements.txt file

Reply:
![image](https://github.com/user-attachments/assets/233618bb-9c51-476f-a53f-c3562d61c0ac)